js/core/config.js

```diff
@@ -80,6 +80,47 @@ const config = {
     */
     useLegacyVisibleIndex: false,
 
+    /**
+    * @name globalConfig.floatingActionButtonConfig
+    * @type object
+    */
+    floatingActionButtonConfig: {
+        /**
+        * @name globalConfig.floatingActionButtonConfig.icon
+        * @type string
+        * @default "preferences"
+        */
+        icon: "add",
+
+        /**
+        * @name globalConfig.floatingActionButtonConfig.closeIcon
+        * @type string
+        * @default "close"
+        */
+        closeIcon: "close",
+
+        /**
+        * @name globalConfig.floatingActionButtonConfig.position
+        * @type Enums.PositionAlignment|positionConfig|function
+        * @default "right bottom"
+        */
+        position: {
+            at: "right bottom",
+            my: "right bottom",
+            offset: {
+                x: -16,
+                y: -16
+            }
+        },
+
+        /**
+        * @name globalConfig.floatingActionButtonConfig.maxActionButtonCount
+        * @type number
+        * @default 6
+        */
+        maxActionButtonCount: 6
```

Comment:
Shouldn't it be five? (it's `from 3 to 6` in guidelines but they count a container button)

Reply:
And we include a container button in this count

Reply:
Not sure it's right. A small example:
I set maxActionButtonCount to 7, placed 7 action items in different places in my code but the 7th action item isn't rendered. It's not what I expect.